### ISSUE TITLE
Current website formatting doesn't meet WCAG text accessibility guidelines

### ISSUE THREAD

**Issue by cbiffle** [cucapra/filament#390]
Hi! Noticed another thing on the website: it's using a very thin typeface (Fira Sans) in a mid-gray on a white background, and relatively low contrast code examples. This makes it difficult for me to read, personally, but running Firefox's accessibility inspector over the site also shows that it fails the quantitative WCAG accessibility guidelines.

Increasing the text contrast would make it easier for folks with low vision or lower contrast sensitivity to read. If you've got Firefox, the accessibility inspector can help with this a lot (used it to improve the stylesheet on my own website).

**Comment by rachitnigam** [comment]
Ah, thanks for pointing this out! We can take a look and fix it
